tfx/components/example_gen/big_query_example_gen/executor_test.py

```diff
@@ -64,9 +67,13 @@ def setUp(self):
     # Mock BigQuery result schema.
     self._schema = [
         bigquery.SchemaField('i', 'INTEGER', mode='REQUIRED'),
+        bigquery.SchemaField('i2', 'INTEGER', mode='REPEATED'),
         bigquery.SchemaField('b', 'BOOLEAN', mode='REQUIRED'),
         bigquery.SchemaField('f', 'FLOAT', mode='REQUIRED'),
+        bigquery.SchemaField('f2', 'FLOAT', mode='REPEATED'),
         bigquery.SchemaField('s', 'STRING', mode='REQUIRED'),
+        bigquery.SchemaField('s2', 'STRING', mode='NULLABLE'),
```

Comment:
REPEATED?

Reply:
I'm afraid it can actually be NULLABLE as well. [Example](https://github.com/tensorflow/tfx/pull/1731#discussion_r421064061)

Reply:
In the current implementation is however ignored so it doesn't matter, but future solutions should be aware of this.

Reply:
for this test, since 's2': ['abc', 'def'], it should be REPEATED? or you mean it might be None here?

Reply:
Exactly! It is not NULL, but it is NULLABLE as in could be NULL. See example [here](https://github.com/tensorflow/tfx/pull/1731#discussion_r421064061).

Reply:
NULLABLE means 's2' could be missing or 's2' could be an empty list

Reply:
for 's2': ['abc', 'def'], it is REPEATED
for 's2': "['abc', 'def']", it might be REQUIRED or NULLABLE

Reply:
Sorry for being so slow. You're correct, my example don't actually say anything about NULLABLE or REPEATED. If I had studied [FORMAT](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#format_string) more closely, rather than just copy-pasted it, it actually just converts any format into a string. 

If you replace the example query with `SELECT CAST(NULL AS ARRAY<String>) AS x UNION ALL
SELECT ['str1', 'str2'] AS x`
SchemaField('x', 'STRING', 'REPEATED', None, ())

This proves that it is REPEATED eventhough it is NULL. I'll update it!